src/files/MMTFFile.cpp

```diff
@@ -11,6 +11,10 @@ using namespace chemfiles;
 MMTFFile::MMTFFile(std::string filename, File::Mode mode):
   File(std::move(filename), mode), handle_(MMTF_container_new()) {
 
+    #ifdef __EMSCRIPTEN__
```

Comment:
I would rather put the registration in `TrajectoryFactory` inside ifdef, it feels clearer.